lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -39,6 +39,9 @@ sub index {
         push(@results, $build_results) if $build_results;
     }
     $self->stash('results', \@results);
+    $self->respond_to(
+        json => {json     => {results => \@results}},
```

Comment:
This line does the magic. As far as I see it this is the only line the current tests don't cover (beside the corresponding line for the group overview).